layers/+tools/xclipboard/README.org

```diff
@@ -0,0 +1,15 @@
+#+TITLE: xclipboard layer
+
+* Table of Contents                                        :TOC_4_gh:noexport:
+ - [[#description][Description]]
+ - [[#key-bindings][Key Bindings]]
+
+* Description
+This layer adds copy/paste support to the X-clipboard from the terminal.
+
+* Key Bindings
+
+| Key Binding | Description                                 |
+|-------------+---------------------------------------------|
+| ~SPC o y~   | Copy selection to clipboard                 |
+| ~SPC o p~   | Paste clipboard contents at cursor position |
```

Comment:
Fixed to `SPC x y` and `SPC x p`.

As for making normal evil copy/paste commands work like this: I'm not familiar enough with elisp to know how to do that, since interprogram-copy-function behaves differently than expected as reported by @sdwolf.